packages/cozy-procedures/src/components/Documents.jsx

```diff
@@ -58,7 +58,7 @@ class Documents extends React.Component {
             const filesStatusByCategory = filesStatus[categoryId]
             return (
               <section key={index}>
-                <Label>
+                <Label htmlFor="">
```

Comment:
I think this shows that `htmlFor` should not be a required prop on `Label`. What do you think?

Reply:
Or that we shouldn't use `Label` since `label` should be use with `input`.

Reply:
We should use `<Caption>` or something like that instead.

Reply:
OK :+1:

Reply:
After reading the HTML spec, `for` is not mandatory, so let's remove the required